tests/integration/shutdown.tcl

```diff
@@ -0,0 +1,157 @@
+foreach how {sigterm shutdown} {
+    test "Shutting down master waits for replica to catch up ($how)" {
+        start_server {} {
+            start_server {} {
+                set master [srv -1 client]
+                set master_host [srv -1 host]
+                set master_port [srv -1 port]
+                set master_pid [srv -1 pid]
+                set replica [srv 0 client]
+                set replica_pid [srv 0 pid]
+
+                # Config master.
+                $master config set repl-diskless-sync yes
+                $master config set repl-diskless-sync-delay 1
```

Comment:
why not 0?

Reply:
don't know :-) I guess I though it might have some special meaning... I'll use 0.